fixtures/flow-annotations.js.sem.uast

```diff
@@ -416,17 +416,159 @@
                         ],
                         static: false,
                      },
+                     { '@type': "ClassMethod",
+                        '@role': [Declaration, Function, Statement],
+                        '@pos': { '@type': "uast:Positions",
+                           start: { '@type': "uast:Position",
+                              offset: 219,
+                              line: 7,
+                              col: 5,
+                           },
+                           end: { '@type': "uast:Position",
+                              offset: 367,
+                              line: 10,
+                              col: 10,
+                           },
+                        },
+                        async: false,
+                        body: { '@type': "uast:Block",
+                           '@role': [Body, Function, Value],
+                           '@pos': { '@type': "uast:Positions",
+                              start: { '@type': "uast:Position",
+                                 offset: 365,
+                                 line: 10,
+                                 col: 8,
+                              },
+                              end: { '@type': "uast:Position",
+                                 offset: 367,
+                                 line: 10,
+                                 col: 10,
+                              },
+                           },
+                           Statements: [],
+                        },
+                        computed: false,
+                        generator: false,
+                        id: ~,
+                        key: { '@type': "uast:Identifier",
+                           '@role': [Key, Name],
+                           '@pos': { '@type': "uast:Positions",
+                              start: { '@type': "uast:Position",
+                                 offset: 219,
+                                 line: 7,
+                                 col: 5,
+                              },
+                              end: { '@type': "uast:Position",
+                                 offset: 233,
+                                 line: 7,
+                                 col: 19,
+                              },
+                           },
+                           Name: "testfnc_object",
```

Comment:
I can't find annotations in the semantic tree, it seems like some transformation stage cuts them out.

Reply:
Yes, should I open an issue?

Reply:
It exists in the native tree, so something is wrong with current annotations. Do you want to open a separate issue/PR for it?

Reply:
If you check the PR the annotations I added were really pretty simple of the "just add roles to these types" kind. Probably we should just close this PR since it's incomplete anyway because of #28.

Reply:
Agree, if by "close" you mean "merge"

Reply:
Yeah, can't be merged (more annotations and test won't hurt) but removing the "Fixes" since as soon as it's merged @egorbu will find some other Unannotated from some other file until we fix #28.